src/tests/unit/tests/bug-filing/github/github-bug-filing-service.test.tsx

```diff
@@ -37,15 +37,25 @@ describe('GithubBugFilingServiceTest', () => {
     });
 
     test('isSettingsValid', () => {
-        const emptySettings: GitHubBugFilingSettings = null;
-        const invalidSettings: GitHubBugFilingSettings = {
+        const nullSettings: GitHubBugFilingSettings = null;
```

Comment:
does it make sense to use `it.each` to separate all this scenarios now?

Reply:
updated